src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -221,6 +225,32 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
                 getBrowserWindow().globalStores.routing.updateRoute(newParams, undefined, true, true);
             }
         );
+        
+        if (this.props.replaceRoutingHistoryCallback) {
+            this.silentUrlParamsReaction = reaction(
+                ()=>[
+                    this.sortMode
+                ],
+                ()=>{
+                    const regex = new RegExp('&' + ONCOPRINT_SORTBY_URL_PARAM + '=[^&]*');
```

Comment:
not important here but just want to make sure you know about es6 template strings which allow for string composition without annoying concatenation.  again, you don't need to update this.

Reply:
just to be clear, you do NOT need to update this